docs/en/week06/06-3.md

```diff
@@ -0,0 +1,203 @@
+
+# Deep Learning Lab 6: RNN 
+
+Authors:
+    - Zhengyuan Ding
+    - Biao Huang
+    - Lin Jiang
+Editor: 
+    - Nhung Le
+
+## Overview (0:00 - 14:00)
+
+RNN is one type of architectures that we can use to deal with sequences of data. What is a sequence? From the CNN lesson we see that a signal can be 1D,2D or 3D based on the domain. The domain is what you map from to go to. Handling sequential data is basically dealing with 1D data since the domain are just temporal input Xs. Nevertheless, you can also use RNN to deal with 2D data, where you have double directions.
+
+#### 1. Vanilla vs. Recurrent NN 
+![Vanilla](https://i.imgur.com/cY4AbHu.png)
```

Comment:
Include the actual images, not links!!
Where's the caption? Check https://atcold.github.io/pytorch-Deep-Learning/en/about/